microsoft-365/bookings/add-staff.md

```diff
@@ -94,7 +100,12 @@ Here's an example PowerShell command to add Allie Bellew to the Contoso daycare
     ```powershell
     Add-MailboxPermission -Identity "daycare@contoso.com" -User "Allie Bellew" -AccessRights FullAccess -InheritanceType All
     ```
-
+     By default, the mailbox gets automatically mapped to Outlook. If you do not want the booking mailbox to appear in the user's Outlook, assign full access with the following commands:
```

Comment:
```suggestion
     By default, the mailbox gets automatically mapped to Outlook. If you do not want the booking mailbox to appear in the user's Outlook, assign full access with the following commands:
```

Reply:
@JohanFreelancer9 I don't see the option to Commit